llvm/lib/Transforms/Scalar/LICM.cpp

```diff
@@ -2249,7 +2249,7 @@ bool llvm::promoteLoopAccessesToScalars(
     if (SawUnorderedAtomic)
       PreheaderLoad->setOrdering(AtomicOrdering::Unordered);
     PreheaderLoad->setAlignment(Alignment);
-    PreheaderLoad->setDebugLoc(DebugLoc());
+    PreheaderLoad->dropLocation();
```

Comment:
Why the change to `dropLocation`?

Reply:
I think this is a mistake on my part, along with replacing `dropLocation()` with `DebugLoc::getDropped()` above - it should be the other way around (i.e. this uses `getDropped()` and the above keeps using `dropLocation()`).